server/gateway/src/odspUtils.ts

```diff
@@ -36,11 +41,73 @@ export function isSpoServer(server: string) {
     return false;
 }
 
+// TODO: These functions are taken from @fluidframework/odsp-utils package and should be removed
```

Comment:
Can we open an issue for this and assign it?

Reply:
Created #3871 to track